tools/github/scripts/Validate-AzsdkCodeOwner.ps1

```diff
@@ -16,8 +16,8 @@ $hasPermissions = $false
 $orgResponse = (gh api "https://api.github.com/users/$UserName/orgs")
 $orgs = $orgResponse | ConvertFrom-Json
 
-if ($orgs -ne $null) {
-    $orgs = $orgs | select -Expand login
+if (!$org) {
```

Comment:
Doesn't this reverse the logic?   Previously, it was a 'not null" check, now it's a (forgiving) null check?